code/sorting/src/bogo_sort/bogo_sort.c

```diff
@@ -7,29 +7,17 @@
 
 #include <stdio.h>    
 #include <stdlib.h>    
-
-int
-main()    
-{        
-    int num[10]={1, 4, 7, 5, 9, 2, 6, 3, 8, 0};        
-    int i;            
-    bogosort(num, 10);        
-    printf("The array after sorting is:");        
-    for (i = 0;i < 10;i++) {            
-        printf("%d\n", num[i]);        
-    } printf("\n");    
-}         
-int
-is_sorted(int *a, int n)    
+        
+int is_sorted(int *a, int n)    

```

Comment:
The type goes on is own line. See https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c